Gemfile.lock

```diff
@@ -163,7 +172,7 @@ GEM
     kgio (2.10.0)
     launchy (2.4.3)
       addressable (~> 2.3)
-    libv8 (3.16.14.13)
+    libv8 (3.16.14.19)
```

Comment:
@lyrahall14 When I set this up on my machine (running High Sierra), all I needed to do to get it working is to make this lone change (swapping 13 => 19) and nothing else.  Successfully `bundle install` after making the lockfile change.  Could you try removing the other changes, merging in the latest changes from `master`, and seeing if it'll pass the tests?

Reply:
The PR I just merged (#297) contains the above change and for me also fixed this issue under High Sierra.

Reply:
👍 thanks! closing this issue now then